sbt-scala-native/src/main/scala/scala/scalanative/sbtplugin/ScalaNativePluginInternal.scala

```diff
@@ -221,8 +221,7 @@ object ScalaNativePluginInternal {
             val isCppSource = path.endsWith(".cpp")
 
             val compiler = abs(if (isCppSource) clangpp else clang)
-            val flags = Seq("-O2") ++ (if (isCppSource) Seq("-std=c++11")
-                                       else Seq()) ++ opts
+            val flags    = (if (isCppSource) Seq("-std=c++11") else Seq()) ++ opts
```

Comment:
We do want to keep `-O2` as code from runtime library should always be optimized, even in debug mode.